src/spec/create_extension_spec.py

```diff
@@ -0,0 +1,531 @@
+                    "+ is up. Coordinate is relative to the zero-point described in `position_reference`."
+                ),
+                dtype="float",
+                quantity="?",
+            ),
+            NWBDatasetSpec(
+                name="estimated_brain_area",
+                neurodata_type_inc="VectorData",
+                dtype="text",
+                doc=('The brain area of the estimated contact position, e.g., "CA1".'),
+                quantity="?",
+            ),
+            NWBDatasetSpec(
+                name="confirmed_position_ap_in_mm",
+                neurodata_type_inc="VectorData",
+                doc=(
+                    "Anteroposterior (AP) stereotactic coordinate of the confirmed contact position, in millimeters. "
+                    "+ is anterior. Coordinate is relative to the zero-point described in `position_reference`."
+                ),
+                dtype="float",
+                quantity="?",
+            ),
+            NWBDatasetSpec(
+                name="confirmed_position_ml_in_mm",
+                neurodata_type_inc="VectorData",
+                doc=(
+                    "Mediolateral (ML) stereotactic coordinate of the confirmed contact position, in millimeters. "
+                    "+ is right. Coordinate is relative to the zero-point described in `position_reference`."
+                ),
+                dtype="float",
+                quantity="?",
+            ),
+            NWBDatasetSpec(
+                name="confirmed_position_dv_in_mm",
+                neurodata_type_inc="VectorData",
+                doc=(
+                    "Dorsoventral (DV) stereotactic coordinate of the confirmed contact position, in millimeters. "
+                    "+ is up. Coordinate is relative to the zero-point described in `position_reference`."
+                ),
+                dtype="float",
+                quantity="?",
+            ),
+            NWBDatasetSpec(
+                name="confirmed_brain_area",
+                neurodata_type_inc="VectorData",
+                dtype="text",
+                doc=('The brain area of the actual contact position, e.g., "CA1".'),
+                quantity="?",
+            ),
+        ],
+        links=[
+            NWBLinkSpec(
+                name="probe",
+                doc="The probe that the channels belongs to.",
+                target_type="Probe",
+            ),
+        ],
+        attributes=[
+            NWBAttributeSpec(
+                name="position_reference",
+                doc=(
+                    "Location of the origin (0, 0, 0) for `{X}_position_{Y}_in_mm` coordinates, e.g., "
+                    '"(AP, ML, DV) = (0, 0, 0) corresponds to bregma at the cortical surface".'
+                ),
+                dtype="text",
+                required=False,
+            ),
+            NWBAttributeSpec(
+                name="reference_mode",
```

Comment:
Given that we are thinking on adding `position_reference` could we change this to `electrical_reference_mode` or something of the sort? What do you guys think? I think is more clear even if `position_refrence` did not exist.

Reply:
I like `electrical_reference_mode`. I will make that change

Reply:
Great